include/MFAnalog.h

```diff
@@ -37,5 +40,10 @@ class MFAnalog
     uint32_t      _last;
     analogEvent   _handler; 
     uint8_t       _sensitivity;
+
+    uint16_t ADC_Buffer[ADC_MAX_AVERAGE] = {0};     // Buffer for all values from each channel  
+    uint16_t ADC_Average_Buffer = 0;                // sum of sampled values, must be divided by ADC_MAX_AVERAGE to get actual value
```

Comment:
How about calling it `ADC_Running_Total`?

Reply:
Yeah. Don't know why it selected three lines. Weird.

Reply:
What would be about a combination: `ADC_Average_Total`

Reply:
Done.